### DISCUSSION TITLE
Api routes or Route handlers for API?

### DISCUSSION THREAD

**Discussion by michele-gobbi** [vercel/next.js#61457] [App Router]
I'm going to use Nextjs 14 for creating few simple API endpoints. 
I'm new in Next.js so I am asking if I have to choose the **Api Routes** or the **Route Handlers** to create my endpoints.

They seem to be two different approch to reach the same result. Which is the best one? and the modern one?
Thanks

**Comment by icyJoseph** [comment]
If you want the latest thing, go with Route Handlers, which are available in the App Router tech stack.

**Reply by michele-gobbi** [reply]
Thank you!
